### PR TITLE
quiet dependency installs

### PR DESCRIPTION
**Pull request by TomAugspurger**
Cleans up the CI logs a bit.

### CODE REVIEW

File: continuous_integration/appveyor/setup_conda_environment.cmd [hunk 1]
```diff
@@ -20,14 +20,14 @@ conda create -n %CONDA_ENV% -q -y python=%PYTHON% pytest toolz
...
-%CONDA_INSTALL% -c conda-forge "numpy>=1.13" "pandas>=0.21.0" cloudpickle distributed bcolz bokeh h5py ipython lz4 psutil pytables s3fs scipy fastparquet python-snappy zarr crick
+%CONDA_INSTALL% -c conda-forge -q "numpy>=1.13" "pandas>=0.21.0" cloudpickle distributed bcolz bokeh h5py ipython lz4 psutil pytables s3fs scipy fastparquet python-snappy zarr crick
 
-%PIP_INSTALL% --no-deps --upgrade locket git+https://github.com/dask/partd
-%PIP_INSTALL% --no-deps --upgrade heapdict git+https://github.com/dask/cachey
-%PIP_INSTALL%           --upgrade git+https://github.com/dask/distributed
-%PIP_INSTALL% --no-deps           git+https://github.com/pydata/sparse
-%PIP_INSTALL% --no-deps --upgrade blosc --upgrade
-%PIP_INSTALL% --no-deps moto Jinja2 boto boto3 botocore cryptography requests xmltodict six werkzeug PyYAML pytz python-dateutil python-jose mock docker jsondiff==1.1.2 aws-xray-sdk responses idna cfn-lint
+%PIP_INSTALL% -- quiet --no-deps --upgrade locket git+https://github.com/dask/partd
```
Comment (jrbourbeau) [inline continuous_integration/appveyor/setup_conda_environment.cmd]:
```suggestion
%PIP_INSTALL% --no-deps --upgrade locket git+https://github.com/dask/partd
```

File: continuous_integration/appveyor/setup_conda_environment.cmd [hunk 2]
```diff
@@ -20,14 +20,14 @@ conda create -n %CONDA_ENV% -q -y python=%PYTHON% pytest toolz
 call activate %CONDA_ENV%
 
 @rem Install optional dependencies for tests
-%CONDA_INSTALL% -c conda-forge "numpy>=1.13" "pandas>=0.21.0" cloudpickle distributed bcolz bokeh h5py ipython lz4 psutil pytables s3fs scipy fastparquet python-snappy zarr crick
+%CONDA_INSTALL% -c conda-forge -q "numpy>=1.13" "pandas>=0.21.0" cloudpickle distributed bcolz bokeh h5py ipython lz4 psutil pytables s3fs scipy fastparquet python-snappy zarr crick
```
Comment (jrbourbeau) [inline continuous_integration/appveyor/setup_conda_environment.cmd]:
```suggestion
%CONDA_INSTALL% -c conda-forge "numpy>=1.13" "pandas>=0.21.0" cloudpickle distributed bcolz bokeh h5py ipython lz4 psutil pytables s3fs scipy fastparquet python-snappy zarr crick
```

**Comment by jrbourbeau** [comment]
Pushed a commit to address my minor comment re: Appveyor, hope that okay

**Comment by TomAugspurger** [comment]
Much appreciated, thanks.

**Comment by jrbourbeau** [comment]
Will merge later this evening if no other responses

**Comment by jrbourbeau** [comment]
Merging, thanks @TomAugspurger!

**Pull request merged**